server/src/test/java/keywhiz/service/crypto/ContentCryptographerTest.java

```diff
@@ -46,7 +45,7 @@
   }
 
   @Before public void setUp() throws Exception {
-    cryptographer = new ContentCryptographer(BASE_KEY, new SunJCE(), BC, FakeRandom.create());
+    cryptographer = new ContentCryptographer(BASE_KEY, BC, BC, FakeRandom.create());
```

Comment:
You can also `Security.getProvider("SunJCE")` if you want SunJCE, instead of calling the constructor

Reply:
I thought that didn't work anymore either, but @csstaub tells me I'm wrong.

Either way, we've got a BC here, so no harm in using it in this test, I think?